README.md

```diff
@@ -76,6 +76,11 @@ Connections between clients and servers are established through transports like
 MCP servers follow a decorator approach to register handlers for MCP primitives like resources, prompts, and tools. The goal is to provide a simple interface for exposing capabilities to LLM clients.
 
 ```python
+# /// script
+# dependencies = [
+#   "mcp"
+# ]
+# ///
```

Comment:
The inline script dependencies trick here also means you don't even have to install `mcp` separately. See: https://simonwillison.net/2024/Nov/24/follow-theirs/

Reply:
TIL. Thank you